### ISSUE TITLE
bug(schema-error-property-style): rule should report invalid esQuery selectors

### ISSUE THREAD

**Issue by marcalexiei** [marcalexiei/eslint-plugin-zod-x#175]
### 🧩 Description

As title

### 🪜 How to reproduce

Ruel config

```ts
{
  'zod-x/schema-error-property-style': ['error', { selector: '!>asd' }],
}
````

```ts
import * as z from 'zod';

z.custom(() => 'asd', `asd`);
z.custom(() => true, { error: () => 'my error' });
```

### ✅ Expected behavior

Error should inform the user that an esquery selector is invalid

### 💻 System info

```shell
Not relevant
```
